packages/react-server/src/forks/ReactServerStreamConfig.dom-fb-experimental.js

```diff
@@ -42,6 +42,19 @@ export interface Destination {
   onError(error: mixed): void;
 }
 
+function handleErrorInNextTick(error: any) {
+  setTimeout(() => {
+    throw error;
+  });
+}
+
+export const scheduleMicrotask: (callback: () => void) => void =
+  typeof queueMicrotask === 'function'
+    ? queueMicrotask
+    : callback => {
+        Promise.resolve(null).then(callback).catch(handleErrorInNextTick);
+      };
+
```

Comment:
Not sure about Meta's plans here. Similar to the legacy case above the schedule work is sync (why?) so the microtask scheduler should probably also be sync. Unlike with legacy I can't error b/c flight is supported here. 

That said, sync isn't really desirable outside the legacy case for renderToString so I think we should just make scheduleWork use whatever schedule is preferable for Meta